balloon.py

```diff
@@ -238,3 +239,56 @@ def balloon_m_hash(password: str, salt: str) -> str:
     return balloon_m(
         password, salt, space_cost, time_cost, parallel_cost, delta=delta
     ).hex()
+
+
+def verify(
+    password: str, salt: str, space_cost: int, time_cost: int, delta: int, hash: str
```

Comment:
Can we put `hash` at the start?
We can then also make `delta` 3 by default again.